native/src/components/MapView.tsx

```diff
@@ -37,126 +37,126 @@ const featureLayerId = 'point'
 
 // Has to be set even if we use map libre
 MapboxGL.setAccessToken(mapConfig.accessToken)
-const MapView = ({
-  boundingBox,
-  featureCollection,
-  selectedFeature,
-  fabPosition,
-  onRequestLocationPermission,
-  locationPermissionGranted,
-  selectPoiFeature,
-  setSheetSnapPointIndex
-}: MapViewPropsType): ReactElement => {
-  const [followUserLocation, setFollowUserLocation] = useState<boolean>(false)
-  const mapRef = React.useRef<MapboxGL.MapView | null>(null)
-  const cameraRef = React.useRef<MapboxGL.Camera | null>(null)
-  const theme = useTheme()
-
-  const layerProps: SymbolLayerProps = {
-    id: featureLayerId,
-    style: {
-      symbolPlacement: 'point',
-      symbolZOrder: 'source',
-      iconAllowOverlap: true,
-      iconSize: mapMarker.iconSize,
-      iconIgnorePlacement: true,
-      iconImage: [
-        'case',
-        ['==', ['get', 'id'], selectedFeature?.properties.id ?? -1],
-        mapMarker.symbolActive,
-        ['get', 'symbol']
-      ],
-      textField: ['get', 'title'],
-      textFont: ['Roboto Regular'],
-      textOffset: [0, textOffsetY],
-      textAnchor: 'top',
-      textSize: 12
+const MapView = React.forwardRef(
```

Comment:
Am I correct that the only thing changed here is the wrapping with `forwardRef`? Why is it needed to call `cameraRef.flyTo` in the parent component?

Reply:
Well we want to use the `selectPoiFeature` in the MapView and ListItem. This method uses the cameraRef that has to be set in MapView(Camera) to gain control to the Camera Component. 
So i think its the easiest way to pass the `selectedPoiFeature` fkt from POIS to the child component